docs: known issues 5.1 (#54446)
sourcegraph/sourcegraph: CHANGELOG.md

> **Note**: As of 5.1.0, Sourcegraph OSS is removed and has been relicensed to Sourcegraph Enterprise. A blog post with much more detail will come soon!

### Known issues

> - There is an issue with repository permissions not being enforced on code hosts that do not support the `authorization` field.

Author: There is an issue with repository permissions not being enforced on Sourcegraph instances that are configured to use both explicit permissions and code hosts that do not support the `authorization` field.

> - There is an issue with repository permissions not being enforced on code hosts that do not support the `authorization` field.

Author: ```suggestion
There is an issue with repository permissions not being enforced on Sourcegraph instances that are configured to use both explicit permissions and code hosts that do not support the authorization field.
```

> There is an issue with repository permissions not being enforced on Sourcegraph instances that are configured to use both explicit permissions and code hosts that do not support the authorization field. Customers using the explicit permissions API are advised to upgrade to v5.1.1 directly.

Author: ```suggestion
There is an issue with Sourcegraph instances configured to use explicit permissions using permissions.userMapping in Site configuration, where repository permissions are not enforced. Customers using the explicit permissions API are advised to upgrade to v5.1.1 directly.
```
